server/src/main/java/org/apache/iotdb/db/metadata/mnode/InternalMNode.java

```diff
@@ -169,6 +169,12 @@ public synchronized void replaceChild(String oldChildName, IMNode newChildNode)
       newChildNode.getAsEntityMNode().setUseTemplate(oldChildNode.isUseTemplate());
     }
 
```

Comment:
Just be curious, why do we need to clone the children from `oldChildNode` to `newChildNode`...

Reply:
We define the parent node of measurementMNode as Device/Entity, thus there will be a MNode type change when creating timeseries under an existing InternalMNode.